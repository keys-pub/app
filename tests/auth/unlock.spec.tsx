import * as React from 'react'
import * as renderer from 'react-test-renderer'

import AuthLock from '../../src/renderer/views/auth/unlock'

describe('Auth unlock', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<AuthLock />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
