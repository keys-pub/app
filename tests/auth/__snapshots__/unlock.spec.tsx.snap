// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth unlock renders correctly 1`] = `
<div
  className="MuiBox-root MuiBox-root-1"
  style={
    Object {
      "height": "100%",
    }
  }
>
  <div
    className="MuiBox-root MuiBox-root-2"
    style={
      Object {
        "width": "100%",
      }
    }
  >
    <div
      className="MuiBox-root MuiBox-root-3"
      style={
        Object {
          "height": 28,
        }
      }
    >
      <div
        className="MuiBox-root MuiBox-root-4"
      >
        <div
          className="MuiBox-root MuiBox-root-5 drag"
        />
        <div
          className="MuiBox-root MuiBox-root-6"
          style={
            Object {
              "height": 41,
              "marginBottom": -2,
              "marginTop": -6,
            }
          }
        />
      </div>
    </div>
  </div>
  <div
    className="MuiBox-root MuiBox-root-7"
    style={
      Object {
        "paddingBottom": 20,
        "width": "100%",
      }
    }
  >
    <div
      className="MuiBox-root MuiBox-root-8"
      style={
        Object {
          "height": 60,
        }
      }
    />
    <div
      className="MuiBox-root MuiBox-root-9"
      style={
        Object {
          "alignSelf": "center",
        }
      }
    >
      <p
        className="MuiTypography-root MuiTypography-body1"
        style={
          Object {
            "color": "#2196f3",
            "fontFamily": "Minercraftory",
            "fontSize": 64,
          }
        }
      >
        keys
      </p>
    </div>
    <hr
      className="MuiDivider-root"
      style={
        Object {
          "marginBottom": 3,
        }
      }
    />
  </div>
  <p
    className="MuiTypography-root MuiTypography-body1"
    style={
      Object {
        "paddingBottom": 20,
        "paddingTop": 10,
      }
    }
  >
    Enter your password to unlock.
  </p>
  <div
    className="MuiFormControl-root"
    style={
      Object {
        "marginBottom": 10,
      }
    }
  >
    <div
      className="MuiFormControl-root MuiTextField-root"
      style={
        Object {
          "fontSize": 48,
          "width": 400,
        }
      }
    >
      <label
        className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
        data-shrink={false}
      >
        Password
      </label>
      <div
        className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          autoFocus={true}
          className="MuiInputBase-input MuiOutlinedInput-input"
          disabled={false}
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          required={false}
          style={
            Object {
              "fontSize": 32,
              "height": 18,
            }
          }
          type="password"
          value=""
        />
        <fieldset
          aria-hidden={true}
          className="PrivateNotchedOutline-root-10 MuiOutlinedInput-notchedOutline"
        >
          <legend
            className="PrivateNotchedOutline-legendLabelled-12"
          >
            <span>
              Password
            </span>
          </legend>
        </fieldset>
      </div>
    </div>
    <p
      className="MuiFormHelperText-root"
      id="component-error-text"
    >
      <span
        dangerouslySetInnerHTML={
          Object {
            "__html": "&#8203;",
          }
        }
      />
    </p>
  </div>
  <div
    className="MuiBox-root MuiBox-root-14"
  >
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-outlinedSizeLarge MuiButton-sizeLarge"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        Unlock
      </span>
    </button>
  </div>
</div>
`;
